### ISSUE TITLE
List Filters (by default should be off) or have a possibility to be in elimination-mode 

### ISSUE THREAD

**Issue by theodorDiaconu** [bluelibs/bluelibs#57]

**Comment by theodorDiaconu** [comment]
@Fen747 while I love this idea of differentiating from defaults I realise that if you want a specific designed way to work with your app, you can always go into shortcuts, and change field in such a way that you "deep-merge" it however you want, for example:

instead of having
```ts
collection({
   id: "xxx",
   fieldDefaults: { ... }
});
```

I think it's more elegant to go to shortcuts, re-export the `field()` and set your own because most of the times you would like that behavior for all collections eliminating repetition or then having `fieldDefaults` at app level too, that's the advantage of having our 'schema' functional. 

And if you want it collection only, you can set your 'field' function inside the collection file.

**Comment by theodorDiaconu** [comment]
TL;DR decided to avoid API configuration polution
